### PR TITLE
Revert "Revert "Feature - CartoDB sync admin feature""

### PR DESCRIPTION
**Pull request by rkrupinski**
Reverts unicef/etools-partner-reporting-portal#309

🐸

### CODE REVIEW

**Comment by rkrupinski** [comment]
@n1207n Please merge `develop` to `revert-309-revert-299-feat-cartodb-sync-265` first.

cc @dfryta

**Comment by n1207n** [comment]
Fixed CartoDB sync function without intervention link, as that is removed from Location.

I was able to load the location data for Mali.

Had to add @python_2_unicode_compatible decorator to fix UnicodeEncodeError in django-admin, and this is alignment with Python 3, as the decorator will be simply no-op and we can remove it later.

**Comment by rkrupinski** [comment]
@dfryta ^^

File: django_api/django_api/apps/core/models.py [hunk 1]
```diff
@@ -128,22 +107,22 @@ class ResponsePlan(TimeStampedModel):
...
+class GatewayType(TimeStampedModel):
     """
     Represents an Admin Type in location-related models.
     """
 
     name = models.CharField(max_length=64L, unique=True)
     admin_level = models.PositiveSmallIntegerField()
 
-    intervention = models.ForeignKey(Intervention, related_name="gateway_types")
+    workspace = models.ForeignKey(Workspace, related_name="gateway_types")
```
Comment (kasfaw) [inline django_api/django_api/apps/core/models.py]: @n1207n gatewaytypes will be unique by country, not workspace

File: django_api/django_api/apps/core/admin.py [hunk 2]
```diff
@@ -1,22 +1,90 @@
...
+    #             fields.append('point')
+    #         if obj.geom:
+    #             fields.append('geom')
+    #
+    #     return fields
+
+
 class GatewayTypeAdmin(admin.ModelAdmin):
     form = GatewayTypeModelForm
     fields = ('name', 'admin_level', 'intervention')
```
Comment (kasfaw) [inline django_api/django_api/apps/core/admin.py]: @n1207n do you mean `fields = ('name', 'admin_level', 'country')`

File: django_api/django_api/apps/core/forms.py [hunk 3]
```diff
@@ -14,5 +81,5 @@ class Meta:
         fields = (
             'name',
             'admin_level',
-            'intervention',
+            'workspace',
```
Comment (kasfaw) [inline django_api/django_api/apps/core/forms.py]: @n1207n should be 'country'

File: django_api/django_api/settings/base.py [hunk 4]
```diff
@@ -69,7 +69,13 @@
 
     'rest_framework',
     'django_filters',
+<<<<<<< HEAD
```
Comment (dfryta) [inline django_api/django_api/settings/base.py]: Once again, conflicts!

File: django_api/django_api/settings/base.py [hunk 5]
```diff
@@ -209,6 +215,44 @@
     }
 }
 
+<<<<<<< HEAD
```
Comment (dfryta) [inline django_api/django_api/settings/base.py]: Same here.

**Pull request merged**